test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1294,6 +1294,16 @@ def forward(self, input1, input2, input3):
 
         self.run_test(TraceModel(), (x1, x2, x3), atol=10e-5)
 
+    def test_numpy_T(self):
+        class NumpyTranspose(torch.nn.Module):
+            def forward(self, x):
+                return x.T
+
+        x1 = torch.randn(4, 7)
+        self.run_test(NumpyTranspose(), x1)
```

Comment:
No need for temporary variables
```suggestion
        self.run_test(NumpyTranspose(), torch.randn(4, 7))
```